### PR TITLE
MAS links for MOVMetaEdit

### PR DESCRIPTION
**Pull request by g-maxime**
Signed-off-by: Maxime Gervais <gervais.maxime@gmail.com>

### CODE REVIEW

File: src/MOVMetaEditBundle/Resources/views/Default/index.html.twig [hunk 1]
```diff
@@ -32,9 +32,15 @@
 
             <div class="inner center-xs center-font-xs">
                 <div class="btn-group hidden-xs">
+                    {% if 'macOS' == downloadInfo.name %}
+                    <a class="btn btn-default btn-lg btn-dl" href="https://itunes.apple.com/app/avimetaedit/id1299543806" target="_blank">
```
Comment (GuillaumeRoques) [inline src/MOVMetaEditBundle/Resources/views/Default/index.html.twig]: Link is for AVIMetaEdit not MOVMetaEdit :)

File: src/MOVMetaEditBundle/Resources/views/Download/macTable.html.twig [hunk 2]
```diff
@@ -1,4 +1,8 @@
 <tr>
+    <th rowspan="2" colspan="2"><a href="http://itunes.apple.com/app/avimetaedit/id1299543806">Mac App Store</a></th>
```
Comment (JeromeMartinez) [inline src/MOVMetaEditBundle/Resources/views/Download/macTable.html.twig]: Brocken table.

File: src/MOVMetaEditBundle/Resources/views/Download/macTable.html.twig [hunk 3]
```diff
@@ -1,4 +1,12 @@
 <tr>
+    <th rowspan="2" colspan="2"><a href="http://itunes.apple.com/app/movmetaedit/id1299543806">Mac App Store</a></th>
+    <td class="table-OS" colspan="3">macOS 10.7 to 10.13, x86_64</td>
+</tr>
+<tr>
+    <th><abbr title="Graphical User Interface">GUI</abbr></th>
+    <td colspan="2">Mac App Store is the simplest way to find and download apps for your Mac.<br /><a href="http://itunes.apple.com/app/avimetaedit/id1289778020">Get AVIMetaEdit 1.0.2 from Mac App Store!</a></td>
```
Comment (GuillaumeRoques) [inline src/MOVMetaEditBundle/Resources/views/Download/macTable.html.twig]: Link and text still for AVIMetaEdit

File: src/MOVMetaEditBundle/Resources/views/Download/macTable.html.twig [hunk 4]
```diff
@@ -1,4 +1,12 @@
 <tr>
+    <th rowspan="2" colspan="2"><a href="http://itunes.apple.com/app/movmetaedit/id1299543806">Mac App Store</a></th>
+    <td class="table-OS" colspan="3">macOS 10.7 to 10.13, x86_64</td>
+</tr>
+<tr>
+    <th><abbr title="Graphical User Interface">GUI</abbr></th>
+    <td colspan="2">Mac App Store is the simplest way to find and download apps for your Mac.<br /><a href="http://itunes.apple.com/app/movmetaedit/id1299543806">Get MOVMetaEdit 17.10 from Mac App Store!</a></td>
+</tr>
+<tr>
```
Comment (JeromeMartinez) [inline src/MOVMetaEditBundle/Resources/views/Download/macTable.html.twig:9]: Missing an empty line as with other macOS DL pages

**Pull request merged**